tests/appsec/appsec/test_remoteconfiguration.py

```diff
@@ -455,483 +475,28 @@ def test_load_new_configurations_remove_config_and_dispatch_applied_configs_erro
     disable_appsec_rc()
 
 
-@mock.patch("ddtrace.appsec._remoteconfiguration._appsec_1click_activation")
-@mock.patch("ddtrace.appsec._remoteconfiguration._appsec_rules_data")
-def test_load_multiple_targets_file_same_product(
```

Comment:
don't we need to tests this scenarios now?

Reply:
This logic is handled in libddwaf now. We don't merge anything anymore.